templates/layout.html

```diff
@@ -307,7 +308,6 @@
 <a href="https://gitter.im/{{ config.THEME_SETTINGS.gitter_room }}" id="open-chat" class="gitter-open-chat-button">Open Chat</a>
 <script defer src="https://sidecar.gitter.im/dist/sidecar.v1.js" type="application/javascript" charset="UTF-8"></script>
 {%- endif %}
-<!-- jQuery -->
```

Comment:
Any reason this comment was removed?

Reply:
ups! It was probably an accident, my bad.

Reply:
Restored in [481a02f](https://github.com/spyder-ide/lektor-icon/pull/26/commits/481a02f78d60147b7af70b2b7da43a18f06dfee2)